Pokemon App/Pokemon App/CardCollectionViewCell/CardCollectionViewCell.swift

```diff
@@ -11,23 +11,35 @@ class CardCollectionViewCell: UICollectionViewCell {
     
     
     @IBOutlet weak var pokemonCardImageView: UIImageView!
+    @IBOutlet weak var imageContainerView: UIView!
+    @IBOutlet weak var errorLabel: UILabel!
+    @IBOutlet weak var errorButton: UIButton!
+    
+    var pokemonImageURL = ""
     
     
     let imageLoader = ImageLoader()
 
     override func awakeFromNib() {
         super.awakeFromNib()
         
+        setUpContainerAndError()
+        
+    }
+    
+    @IBAction func onTapErrorButton(_ sender: Any) {
+        downloadImage()
     }
     
-    func downloadImage(imageURL: String){
-        if let url = URL(string: imageURL) {
+    func downloadImage(){
```

Comment:
```suggestion
    func downloadImage(for url: URL){
```
Ini bisa dirubah jadi parameter aja URLnya, dan `var pokemonImageURL = ""` bisa dihapus. Karena `pokemonImageURL` cuma dipakai di func ini jadi gak perlu dibikin property injection.

Reply:
url nya kepake pas retry mas, atau ada cara lain kah?